src/hooks/useTableSort.ts

```diff
@@ -0,0 +1,26 @@
+import { useState } from 'react';
+
+// This hook handles the state that handles table sorting.
+const useTableSort = () => {
```

Comment:
In later commit this hook will be more complex, because it will handle also the URL change.